contracts/math/uints.ral

```diff
@@ -3,7 +3,7 @@ struct U512 {
     mut lower: U256
 }
 
-Abstract Contract U512Impl () {
+Contract Uints () {
```

Comment:
Please explain the benefits of using a Contract instead of an Abstract Contract

Reply:
abstract contract is used to extend normal contract with functions of abstract contract, it will still produce one contract as an output, bytecode limit of a decimal contract (one where we use uints) was exceeded so i had to split it and make uints as its own contract